app/mesh/1.0.x/installation/amazonlinux.md

```diff
@@ -44,12 +45,24 @@ $ tar xvzf kong-mesh-{{page.kong_latest.version}}*.tar.gz
 {% endnavtab %}
 {% endnavtabs %}
 
-### 2. Follow Kuma Instructions
+### 2. Configure The License
+
+Before running the {{site.mesh_product_name}} control plane process - which is served by the `kuma-cp` executable - we need to make sure that we have a valid {{site.mesh_product_name}} license in place.
+
+This can be done by running the control plane with:
+
+```sh
+$ KUMA_LICENSE_PATH=license.json kuma-cp run
+```
+
+Where `license.json` is the path to a valid {{site.mesh_product_name}} license file on the file system.
+
+### 3. Follow Kuma Instructions
 
 After downloading the {{site.mesh_product_name}} binaries, the remaining
 installation instructions for Kuma are fully compatible with
 {{site.mesh_product_name}}, except you will be running the
 {{site.mesh_product_name}} binaries instead of the vanilla Kuma ones.
 
 To continue the installation, start from the second installation step in
-[the official Kuma installation guide](https://kuma.io/docs/0.7.1/installation/amazonlinux/#_2-run-kuma).
+[the official Kuma installation guide](https://kuma.io/docs/0.7.1/installation/amazonlinux/#_2-run-kuma) keeping in mind that we need to run the control plane with the appropriate license file as described on this page.
```

Comment:
```suggestion
[the official Kuma installation guide](https://kuma.io/docs/0.7.1/installation/amazonlinux/#_2-run-kuma), and include the license from the previous step when running the control plane.
```

Reply:
This is confusing - the previous step already has them run `kuma-cp`. By pointing to step 2 in the linked doc, aren't you telling them to run it twice? At this point, might it make sense to decouple the installation doc for Kong Mesh from Kuma and keep them fully separate, since there isn't much else left after running kuma-cp?

Reply:
Yes it would make tons of sense, but didn't have time to do it.